users/views.py

```diff
@@ -1,11 +1,30 @@
 from django.shortcuts import render, redirect
 from .forms import UserUpdateForm, ProfileUpdateForm
 from django.contrib.auth.decorators import login_required
+from django.views.generic import DetailView
+from users.models import Profile
+from django.contrib.auth.mixins import LoginRequiredMixin
+from posts.models import Post
+
+
+class ProfileDetailView(LoginRequiredMixin, DetailView):
```

Comment:
This class is for showing a list of all the User's posts?
or a specific Post?

Reply:
This class is for showing the profile when entering '/users/<USERNAME>/' URL.

Also this class if from #128

Reply:
Got it